openedx/api_test.py

```diff
@@ -93,6 +96,12 @@ def test_create_edx_user(user, settings, application, access_token_count):
         json=dict(success=True),
         status=status.HTTP_200_OK,
     )
+    responses.add(
+        responses.POST,
+        settings.OPENEDX_API_BASE_URL + OPENEDX_VALIDATION_REGISTRATION_PATH,
+        json={"validation_decisions": {"username": ""}},
+        status=status.HTTP_200_OK,
+    )
```

Comment:
We are adding the same thing in responses, right here and [below](https://github.com/mitodl/mitxonline/pull/757/files#diff-353bcf6e9efd054b7874f0b78245634f9b837e421e1cd56e457016f4c2763e95R149-R154). Is this duplicate or intentional, Plus if we need the same responses additions it might be better to go for a helper mock function that can be reused.

I think there is the same case [here](https://github.com/mitodl/mitxonline/pull/757/files#diff-353bcf6e9efd054b7874f0b78245634f9b837e421e1cd56e457016f4c2763e95R165-R174) too.